package.json

```diff
@@ -33,14 +39,15 @@
     "@math.gl/proj4": "^3.6.3",
     "@probe.gl/stats": "^4.0.4",
     "@reduxjs/toolkit": "^1.9.5",
-    "deck.gl": "^8.8.3",
+    "apache-arrow": "^13.0.0",
```

Comment:
Is it required. We don't have table formats in this application

Reply:
Yes, there is compilation error without "apache-arrow": "^13.0.0" when I replaced "deck.gl": "^8.8.3" by list of modules   "@deck.gl/XXXX": "^8.9.32",